packages/@react-aria/utils/src/findLastIndex.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/**
+ * Is the reverse of Array.prototype.findIndex() by starting from the end of the
+ * array and finding the first match and returning the index of the found item.
+ */
+export function findLastIndex(array: any[], predicate: (v: number) => boolean): number {
```

Comment:
I created this util function, not sure if any other code will ever use it or if the aria package is the right location.